certbot/_internal/lock.py

```diff
@@ -33,7 +33,7 @@ def lock_dir(dir_path):
     :raises errors.LockError: if unable to acquire the lock
 
     """
-    return LockFile(os.path.join(dir_path, '.certbot.lock'))
+    return LockFile(os.path.join(dir_path, '.certbot._internal.lock'))
```

Comment:
I think we should undo the content changes to this file and the other places `.certbot.lock` was changed into `.certbot._internal.lock`. `sed` was greedy and `.certbot.lock` here refers to the name of the lockfile created on disk.

Reply:
ohhhhhhhhhh that explains it. I was wondering...